xfer.py

```diff
@@ -1032,6 +1032,18 @@ def analyze(transfer_manifest_path: Path) -> None:
             sync_request_start = None
             sync_request = {"files": {}, "transfer_files": set(), "verified_files": {}}
 
+    if len(mismatched_filesizes) > 0:
+        for fname, size in mismatched_filesizes.items():
+            logging.error(
+                "- Mismatched file size for %s: expected %d, got %d on disk",
+                fname,
+                size["expected"],
+                size["got"],
+            )
```

Comment:
Maybe capture the line number of the mismatch as well for debugging purposes?

Reply:
Marked this as resolved early, oops, but this is added now